### ISSUE TITLE
Incorrect warning (unclosed character class) with syntax: regexp

### ISSUE THREAD

**Issue by mnpenner** [hsz/idea-gitignore#184]
Screenshot for proof:

![image](https://cloud.githubusercontent.com/assets/1112998/11968650/53f1aa30-a8c3-11e5-9d19-395265e2baef.png)

My full `.hgignore`:

```
syntax: regexp

# dotfiles
(?<![^/])\.

# folders
^plugins/ezc/
^logs/
^session/
^cache/
^app/[^/]+/csv/
^vendor/
^node_modules/
^bower_components/
(?<![^/])www/reports/
(?<![^/])build/
^www/assets/

# extensions
~$
\.(tar\.gz|orig|pyc|rej|map)$

# special files
^phpunit\.phar$
^nohup.out$
(?<![^/])Thumbs\.db$
^npm-debug\.log$
^typings/.*\.d\.ts$
^Vagrantfile$
^webpack.stats.json$

# should we version these files or not?
(?<![^/])dev-dutch-.*\.xml$
```

The character class *is* closed.

**Comment by hsz** [comment]
@mnpenner Thank you for reporting.

This issue has been already fixed with #166 and it will be available in the next release.

**Comment by mnpenner** [comment]
Oh sorry.. I opened that one too. Had been so long I forgot :D I'll wait for the release. Thanks!
